port_tracking_new.py

```diff
@@ -0,0 +1,134 @@
+# Power Port Detection - By: cwallis
+
+# Use this script to control how your OpenMV Cam captures images for your dataset.
+# You should apply the same image pre-processing steps you expect to run on images
+# that you will feed to your model during run-time.
+
+# If somebody walks in front of the target it will not shoot
+# I am going to pretend it's a safety thing even though it's just because
+# it hates me less this way - Celia
+
+import sensor, image, time, math, pyb
+import frc_pixie
+import frc_can
+from pyb import UART
+
+live = True
+
+pixie = frc_pixie.frc_pixie()
+can = frc_can.frc_can(1)
+
+# Set the configuration for our OpenMV frcCAN device.
+can.set_config(2, 0, 0, 0)
+# Set the mode for our OpenMV frcCAN device.
+can.set_mode(1)
+
+sensor.reset()
+sensor.set_pixformat(sensor.RGB565) # Modify as you like.
+sensor.set_framesize(sensor.QVGA) # Modify as you like.
+sensor.skip_frames(time = 2500)
+
+sensor.set_auto_gain(False)
+sensor.set_auto_whitebal(False)
+sensor.set_auto_exposure(True)
+
+original_exposure = sensor.get_exposure_us()
+sensor.set_auto_exposure(False, int(.20 * original_exposure))
+
+clock = time.clock()
+
+uart = UART(1, 115200)
+
+color = bytearray(3)
+color[0] = 0x00
+color[1] = 0xFF
+color[2] = 0x00
+
+if live == False:
+    img_reader = image.ImageIO("/stream_jan30.bin", "r")
+
+roi = (16, 0, 254, 164)
+
+thresholdsG = [(46, 100, -52, -20, -19, 26)]
+
+while(True):
+    can.update_frame_counter() # Update the frame counter.
+    clock.tick()
+
+    greenBlobs = []
+
+    if live == False:
+        img = img_reader.read(copy_to_fb=True, loop=True, pause=True)
```

Comment:
pretty sure this is out of date - currently should be the format 
img = stream.read(copy_to_fb=True, loop=True, pause=True)